src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -778,6 +778,15 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     helixTimegateWhisper->setMinimumWidth(
         helixTimegateWhisper->minimumSizeHint().width());
 
+    auto *helixTimegateVIPs =
+        layout.addDropdown<std::underlying_type<HelixTimegateOverride>::type>(
+            "Helix timegate /vips behaviour",
+            {"Timegate", "Always use IRC", "Always use Helix"},
+            s.helixTimegateVIPs, helixTimegateGetValue, helixTimegateSetValue,
```

Comment:
```suggestion
            s.helixTimegateVIPs,
            helixTimegateGetValue,  //
            helixTimegateSetValue,  //
            false);
```
heck the formatter

Reply:
Applied in 48873214508426063cb93a88e0c1fb59f365c75a